README.md

```diff
@@ -28,7 +28,35 @@ If you want to develop new content for this collection or improve what is alread
 
 ### Testing with `ansible-test`
 
-TBD
+First, Git clone collection and install it manually:
```

Comment:
Can we create `TESTING.md` with details and then put a link here? This seems overwhelming for a newcomer or a person who is just looking for README.

Reply:
Yes, agreed. Your coment makes me realize that we need to clean up the wiki page and the community guide lines too.